database/models/Review.js

```diff
@@ -0,0 +1,26 @@
+import mongoose, { Types } from 'mongoose';
+
+
+const reviewSchema = new Schema({
+  comment: {
+    type: String,
+    required: false,
+  },
+  date: {
+    type: Date,
+    required: true,
+  },
+  rating: {
+    type: Number,
+    required: true,
+    min: 1,
+    max: 5,
+  },
+  user: {
```

Comment:
createdBy not user
  createdBy:{
        type:Types.ObjectId,
        ref:"User"
    },